StatBot/StatBot/Modules/RiotModule.cs

```diff
@@ -5,6 +5,11 @@ namespace StatBot.Modules
 {    
     public class RiotModule : ModuleBase<SocketCommandContext>
     {
+        /// <summary>
+        /// Echo's back the string after the command.
+        /// </summary>
+        /// <param name="echo">The string to echo.</param>
+        /// <returns></returns>
```

Comment:
LOL you don't needa slap `/// <returns></returns>` if it returns nothing, that's pretty self-explanatory B)

Reply:
It's auto generated, but yes I should take it out